sos/policies/distros/cloudlinux.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) Eduard Abdullin <eabdullin@cloudlinux.com>
```

Comment:
Mail different from commit message is intentional? I expect it does not matter if both mails are correct.

Reply:
Yes, this was done on purpose